include/onnxruntime/core/platform/EigenNonBlockingThreadPool.h

```diff
@@ -1400,12 +1404,12 @@ int CurrentThreadId() const final {
     // State transitions, called only from the thread itself
 
     void SetActive() {
-      std::unique_lock<OrtMutex> lk(mutex);
+      std::lock_guard<OrtMutex> lk(mutex);
```

Comment:
Approved here and below unique_lock->lock_guard (highlighting to confirm approving non-whitespace change)